projects/ion/src/lib/radio/radio.component.scss

```
@@ -0,0 +1,130 @@
+@import '../../styles/index.scss';
+
+.ion-input-radio {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.labelDisabled {
+  cursor: pointer;
+  color: #505566;
+}
+
+.labelDisabled-checked {
+  cursor: pointer;
+  color: #aeb2bd;
+}
+
+.radio {
+  cursor: pointer;
+  appearance: none;
+  outline: none;
+  width: 16px;
```

Acredito que possa utilizar o spacing. 
```suggestion
  width: spacing(2);
```